content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+* Directional and point light shadows
+* Clustered forward rendering
+* Frustum culling
+* Significantly faster sprite rendering with less boilerplate
+* Native WebGL2 support
+* High level custom Materials
+* More powerful shaders: preprocessors, imports, WGSL support
+* Bevy ECS ergonomics and performance improvements. No more `.system()`!
+
+Read on for details!
+
+<!-- more -->
+
+## The New Bevy Renderer
+
+**Bevy 0.6** introduces a brand new modern renderer that is:
+* **Faster**: More parallel, less computation per-entity, more efficient CPU->GPU dataflow, and (soon-to-be-enabled) pipelined rendering
+* **Simpler**: Fewer layers of abstraction, simpler data flow, improved low-level, mid-level, and high-level interfaces, direct wgpu access 
+* **Modular to its core**: Standardized 2d and 3d core pipelines, extensible Render Phases and Views, composable entity/component-driven draw functions, shader imports, extensible and repeatable render pipelines via "sub graphs"
+* **Prettier**: We're releasing the new renderer alongside a number of graphical improvements, such as directional and point light shadows, clustered forward rendering (so you can draw more lights in a scene), and spherical area lights. We also have a ton of new features in development (cascaded shadow maps, bloom, particles, shadow filters, and more!)
+* **Industry Proven**: It takes inspiration from battle tested renderer architectures, such as [Bungie's pipelined Destiny renderer](https://advances.realtimerendering.com/destiny/gdc_2015/Tatarchuk_GDC_2015__Destiny_Renderer_web.pdf). We also learned a lot from (and worked closely with) other renderer developers in the Rust space, namely @aclysma ([rafx](https://github.com/aclysma/rafx)) and @cwfitzgerald ([rend3](https://github.com/BVE-Reborn/rend3)). The New Bevy Renderer wouldn't be what it is without them and I highly recommend checking out their projects!
+
+I promise I'll qualify all of those fluffy buzz words below. I am confident that the New Bevy Renderer will be a rallying point for the Bevy graphics ecosystem and (hopefully) the Rust graphics ecosystem at large. We still have _plenty_ of work to do, but I'm proud of what we have accomplished so far and I'm excited for the future!
+
+![bistro day](bistro_day.png)
+
+### Why build a new renderer?
+
+Before we cover whats new, its worth discussing why we embarked on such a massive effort. The old Bevy Renderer got a number of things right:
+* **Modular render logic** (via the Render Graph)
+* **Multiple backends** (both first and third party)
+* **High level data-driven api**: this made it easy and ergonomic to write custom per-entity render logic
+
+However it also had a number of _significant_ shortcomings:
+
+* **Complex**: The "high level ease of use" came at the cost of significant implementation complexity, performance overhead, and invented jargon. Users were often overwhelmed when trying to operate at any level but "high level".
```

Comment:
I don't want to directly qualify these terms right away as I think it gets "into the weeds" too early.  I just added a note in the "embracing wgpu" section that wgpu is our "low level" api (because we never explicitly tied those terms together). I _think_ thats probably good enough, but if you have specific ideas about how to better introduce them without complicating things too early, let me know!